src/api/controllers/orders-controller.ts

```
@@ -170,31 +165,51 @@ export class OrdersController {
 
-    const totalPrice = orderProducts.reduce((acc, item) => {
-      const product = mergedProductsMap.get(item.id)
+    try {
+      totalPrice = mergedProductsMap.entries().reduce((acc, [, item]) => {
+        const price = Number(item.customProductPrice)
+        const quantity = Number(item.quantity)
 
-      if (product) {
-        return acc + item.price! * product.quantity
-      }
+        if (isNaN(price) || isNaN(quantity)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)